int/config/network.go

```diff
@@ -167,21 +174,21 @@ func (n *NetworkManager) SwitchNetwork(selectedNetworkStr string) error {
 	return nil
 }
 
-//nolint:typecheck,nolintlint
-//go:embed config_network.yaml
-var configData embed.FS
-
-// LoadConfig loads network configurations from an embedded YAML file.
+// LoadConfig loads network configurations from YAML file.
 // Returns the loaded network configurations and any error encountered during loading.
-func LoadConfig() (map[string]NetworkConfig, error) {
-	var networksData map[string]NetworkConfig
+func LoadConfig(configDir string) (map[string]NetworkConfig, error) {
+	networkConfigPath := path.Join(configDir, networkConfigFile)
+	if _, err := os.Stat(networkConfigPath); os.IsNotExist(err) {
+		createDefaultConfig(networkConfigPath)
+	}
```

Comment:
What if it's not a `os.IsNotExist` error?

Reply:
good point, I need to handle this error